test/jdk/jdk/jfr/tool/TestPrintContextual.java

```diff
@@ -277,6 +279,17 @@ private static void span(int depth) {
         span.commit();
     }
 
+    private static void awaitUniqueTimestamp() {
+        Instant timestamp = Instant.now();
+        while (timestamp.equals(Instant.now())) {
+            try {
+                Thread.sleep(1);
+            } catch (InterruptedException e) {
+                // ignore
```

Comment:
Why ignore, though? This would make the thread stuck even if someone actively tries to interrupt it. A common practice in these non-critical methods is to restore the interrupt status and return.

```
} catch (InterruptedException e) {
    Thread.currentThread().interrupt();
    return;
}
```